tests/unit/test_blob.py

```diff
@@ -43,6 +43,12 @@ def _make_one(*args, **kw):
         blob._properties.update(properties)
         return blob
 
+    @staticmethod
+    def _make_one_client(*args, **kw):
+        from google.cloud.storage.client import Client
+
+        return Client(*args, **kw)
```

Comment:
It would be a better strategy to return an autospec'ed `Mock` instance, than to depend on the actual `Client` implementation.  E.g.:

```suggestion
        return mock.create_autospec(Client, instance=True, **kw)
```